account-management-api/src/main/java/uk/gov/di/accountmanagement/entity/SessionState.java

```diff
@@ -0,0 +1,5 @@
+package uk.gov.di.accountmanagement.entity;
+
+public enum SessionState {
```

Comment:
i would strip this back as the session is controlled in the front end app and not backend